app/src/main/res/values/styles.xml

```diff
@@ -178,4 +177,72 @@
     <item name="android:windowIsFloating">false</item>
     <item name="android:backgroundDimEnabled">true</item>
   </style>
+
+  <style name="Heading1" parent="TextView.Common1">
+    <item name="android:fontFamily">sans-serif</item>
+    <item name="android:textSize">24sp</item>
+    <item name="fontFamily">sans-serif</item>
+  </style>
+
+  <style name="Heading2" parent="TextView.Common1">
+    <item name="android:fontFamily">sans-serif-medium</item>
+    <item name="android:textSize">20sp</item>
+    <item name="fontFamily">sans-serif-medium</item>
+  </style>
+
+  <style name="Heading3" parent="TextView.Common1">
+    <item name="android:fontFamily">sans-serif-medium</item>
+    <item name="android:textSize">18sp</item>
+    <item name="fontFamily">sans-serif-medium</item>
+  </style>
+
+  <style name="Heading4" parent="TextView.Common1">
+    <item name="android:fontFamily">sans-serif</item>
+    <item name="android:textSize">18sp</item>
+    <item name="fontFamily">sans-serif</item>
+  </style>
+
+  <style name="Heading5" parent="TextView.Common1">
+    <item name="android:fontFamily">sans-serif-medium</item>
+    <item name="android:textSize">14sp</item>
+    <item name="fontFamily">sans-serif-medium</item>
+  </style>
+
+  <style name="Subtitle1" parent="TextView.Common1">
+    <item name="android:fontFamily">sans-serif</item>
+    <item name="android:textSize">16sp</item>
+    <item name="fontFamily">sans-serif</item>
+  </style>
+
+  <style name="Subtitle2" parent="TextView.Common1">
+    <item name="android:fontFamily">sans-serif</item>
+    <item name="android:textSize">14sp</item>
+    <item name="fontFamily">sans-serif</item>
+  </style>
+
+  <style name="TextFieldLabel" parent="TextView.Common1">
+    <item name="android:fontFamily">sans-serif</item>
+    <item name="android:textSize">12sp</item>
+    <item name="fontFamily">sans-serif</item>
+  </style>
+
+  <style name="Body" parent="TextView.Common1">
+    <item name="android:fontFamily">sans-serif</item>
+    <item name="android:textSize">16sp</item>
+    <item name="fontFamily">sans-serif</item>
+  </style>
+
+  <style name="Caption" parent="TextView.Common1">
+    <item name="android:fontFamily">sans-serif-medium</item>
+    <item name="android:textSize">14sp</item>
+    <item name="fontFamily">sans-serif-medium</item>
+  </style>
```

Comment:
According to #1914 description, this should be `sans-serif` not `sans-serif-medium`